x-pack/plugins/monitoring/public/components/elasticsearch/indices/indices.js

```diff
@@ -4,133 +4,119 @@
+import { EuiMonitoringTable } from '../../table';
+import {
+  EuiLink,
+  EuiPage,
+  EuiPageContent,
+  EuiPageBody,
+  EuiSwitch,
+  EuiSpacer,
+} from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
 
-const filterFields = ['name', 'status'];
 const columns = [
   {
-    title: i18n.translate('xpack.monitoring.elasticsearch.indices.nameTitle', {
+    name: i18n.translate('xpack.monitoring.elasticsearch.indices.nameTitle', {
       defaultMessage: 'Name',
     }),
-    sortKey: 'name',
-    secondarySortOrder: SORT_ASCENDING
+    field: 'name',
+    width: '350px',
+    sortable: true,
+    render: (value) => (
+      <div data-test-subj="name">
+        <EuiLink
+          href={`#/elasticsearch/indices/${value}`}
+          data-test-subj={`indexLink-${value}`}
+        >
+          {value}
+        </EuiLink>
+      </div>
+    ),
   },
   {
-    title: i18n.translate('xpack.monitoring.elasticsearch.indices.statusTitle', {
+    name: i18n.translate('xpack.monitoring.elasticsearch.indices.statusTitle', {
       defaultMessage: 'Status',
     }),
-    sortKey: 'status_sort',
-    sortOrder: SORT_DESCENDING // default sort: red, then yellow, then green
+    field: 'status',
+    sortable: true,
+    render: (value) => (
+      <div title={`Index status: ${value}`}>
+        <ElasticsearchStatusIcon status={value} />&nbsp;
+        {capitalize(value)}
+      </div>
+    )
   },
   {
-    title: i18n.translate('xpack.monitoring.elasticsearch.indices.documentCountTitle', {
+    name: i18n.translate('xpack.monitoring.elasticsearch.indices.documentCountTitle', {
       defaultMessage: 'Document Count',
     }),
-    sortKey: 'doc_count'
+    field: 'doc_count',
+    sortable: true,
+    render: value => (
+      <div data-test-subj="documentCount">
+        {formatMetric(value, LARGE_ABBREVIATED)}
+      </div>
+    )
   },
   {
-    title: i18n.translate('xpack.monitoring.elasticsearch.indices.dataTitle', {
+    name: i18n.translate('xpack.monitoring.elasticsearch.indices.dataTitle', {
       defaultMessage: 'Data',
     }),
-    sortKey: 'data_size'
+    field: 'data_size',
+    sortable: true,
+    render: value => (
+      <div data-test-subj="dataSize">
+        {formatMetric(value, LARGE_BYTES)}
+      </div>
+    )
   },
   {
-    title: i18n.translate('xpack.monitoring.elasticsearch.indices.indexRateTitle', {
+    name: i18n.translate('xpack.monitoring.elasticsearch.indices.indexRateTitle', {
       defaultMessage: 'Index Rate',
     }),
-    sortKey: 'index_rate'
+    field: 'index_rate',
+    sortable: true,
+    render: value => (
+      <div data-test-subj="indexRate">
+        {formatMetric(value, LARGE_FLOAT, '/s')}
+      </div>
+    )
   },
   {
-    title: i18n.translate('xpack.monitoring.elasticsearch.indices.searchRateTitle', {
+    name: i18n.translate('xpack.monitoring.elasticsearch.indices.searchRateTitle', {
```

Comment:
For things like this should the translation I’d b updated?

Reply:
Yup, we should. Thanks for pointing this out

Reply:
Well, I'm not so sure I guess. It's consistent to call it `Title` but still use the `name` key, as seen here: https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/public/components/elasticsearch/ccr_shard/ccr_shard.js#L74

Reply:
For column header any of the `Label`, `Title`, `Name` type is suitable. You can apply any of them.